### DISCUSSION TITLE
css bug ?

### DISCUSSION THREAD

**Discussion by digitalrevisor** [baimard/gestion#165] [General]
This is how gestion looks on our nextcloud. is this just an error happening on our nextcloud, 

nextcloud version is 23 on ubuntu 20.04 LTS and php8.1

![image](https://user-images.githubusercontent.com/59980626/153933104-4f442d98-6b48-469e-99c4-338824217e14.png)

**Comment by baimard** [comment]
Very strange. Have you clear your cache ? 

Version of nextcloud ?

Is the application work ?

**Reply by digitalrevisor** [reply]
yes have also used the disable cache function in chrome ->inspect->network tab
the nextcloud version is 23.0.0 and php version is 8.1

**Comment by baimard** [comment]
Gestion version ?

**Reply by digitalrevisor** [reply]
gestion version is 2.0.4

**Comment by baimard** [comment]
Do you have error in JavaScript ? (F12)

**Reply by baimard** [reply]
I totaly can't reproduce ...

**Reply by digitalrevisor** [reply]
hmm.. then it's most likely a server-side problem I guess.
![image](https://user-images.githubusercontent.com/59980626/154496889-0e45e070-a1ce-442d-bdf9-ba8913cc4509.png)

**Reply by digitalrevisor** [reply]
it looks like there are a lot more issues, but I don't have time to look into it now. I just wanted to confirm that it was not a known or normal issue, we have more issues as well but ill research a bit more when I get the time and come back with what I find out. 
Thanks for the quick replies.

**Reply by baimard** [reply]
how that looks an error in the configuration of your company, is the error happend when you turn in EURO format ? 

or what is your currencies ?

**Comment by digitalrevisor** [comment]
Its ussue both on Euro and Norwegian kroner.



[image: image.png]



Med vennleg helsing

Roger Hopland

Statsautorisert revisor

Autorisert regnskapsfører


Digital Revisor AS


Naustdalsvegen 1B, 6800 Førde | Avdelinger - Bergen | Nordfjordeid | Måløy
*S:* +47 57824090 | *M:* +47 90077351 | *E:* ***@***.*** | *W:*
https://jointlyoffice.com/



tor. 17. feb. 2022 kl. 16:25 skrev Benjamin ***@***.***>:

**Comment by baimard** [comment]
I will investigate thank you.

⁣cordialement,
Benjamin AIMARD​

Le 17 févr. 2022 à 16:39, à 16:39, digitalrevisor ***@***.***> a écrit:
>Its ussue both on Euro and Norwegian kroner.
>
>
>
>[image: image.png]
>
>
>
>Med vennleg helsing
>
>Roger Hopland
>
>Statsautorisert revisor
>
>Autorisert regnskapsfører
>
>
>Digital Revisor AS
>
>
>Naustdalsvegen 1B, 6800 Førde | Avdelinger - Bergen | Nordfjordeid |
>Måløy
>*S:* +47 57824090 | *M:* +47 90077351 | *E:* ***@***.*** | *W:*
>https://jointlyoffice.com/
>
>
>
>tor. 17. feb. 2022 kl. 16:25 skrev Benjamin ***@***.***>:
>
>> how that looks an error in the configuration of your company, is the
>error
>> happend when you turn in EURO format ?
>>
>> or what is your currencies ?
>>
>> —
>> Reply to this email directly, view it on GitHub
>>
><https://github.com/baimard/gestion/discussions/165#discussioncomment-2197624>,
>> or unsubscribe
>>
><https://github.com/notifications/unsubscribe-auth/AOJTWUSLIRQJ6PCYOUDIHNDU3UHPBANCNFSM5OMX63PA>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>>
><https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>>
><https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/baimard/gestion/discussions/165#discussioncomment-2197734
>You are receiving this because you commented.
>
>Message ID:
***@***.***>

**Comment by digitalrevisor** [comment]
Its the latest version.

Med vennleg helsing

Roger Hopland

Statsautorisert revisor

Autorisert regnskapsfører


Digital Revisor AS


Naustdalsvegen 1B, 6800 Førde | Avdelinger - Bergen | Nordfjordeid | Måløy
*S:* +47 57824090 | *M:* +47 90077351 | *E:* ***@***.*** | *W:*
https://jointlyoffice.com/



tor. 17. feb. 2022 kl. 16:39 skrev Digital Revisor ***@***.***>:

**Comment by baimard** [comment]
Look that you have an error in your database.

Do you use postgre sql ?

**Comment by digitalrevisor** [comment]
Hi.
Cashe uis cleared.
Version NC is 23.0.0
My language in nextcloud account is Norwegian.

Its still the same issue on left side bar menu.

Have you tested it on your side.

Med vennleg helsing

Roger Hopland





tor. 17. feb. 2022 kl. 16:39 skrev Digital Revisor ***@***.***>:

**Reply by baimard** [reply]
Do you use postgreSql ?

I totaly can't reproduce :(

**Comment by digitalrevisor** [comment]
From NC system site:

mysql

8.0.28

Størrelse: 1 GB

Med vennleg helsing

Roger Hopland

Statsautorisert revisor

Autorisert regnskapsfører


Digital Revisor AS


Naustdalsvegen 1B, 6800 Førde | Avdelinger - Bergen | Nordfjordeid | Måløy
*S:* +47 57824090 | *M:* +47 90077351 | *E:* ***@***.*** | *W:*
https://jointlyoffice.com/



man. 28. feb. 2022 kl. 11:32 skrev Benjamin ***@***.***>:

**Comment by baimard** [comment]
is it possible to show your database structure of gestion_ ?

**Comment by digitalrevisor** [comment]
See answers beyond.

Still that issue:
[image: image.png]

Med vennleg helsing

Roger Hopland

Statsautorisert revisor

Autorisert regnskapsfører


Digital Revisor AS


Naustdalsvegen 1B, 6800 Førde | Avdelinger - Bergen | Nordfjordeid | Måløy
*S:* +47 57824090 | *M:* +47 90077351 | *E:* ***@***.*** | *W:*
https://jointlyoffice.com/



tir. 15. feb. 2022 kl. 04:52 skrev Benjamin ***@***.***>:

> Very strange. Have you clear your cache ?
>
Yes

> Version of nextcloud ?
>
23.0.2

> Is the application work ?
>
Works, but cant see invoice causing the menu bug.

**Reply by baimard** [reply]
is it possible to show your database structure of gestion_ ?

**Reply by baimard** [reply]
Also contact me by mail [benjamin](mailto:benjamin@cybercorp.fr)

**Comment by baimard** [comment]
Do you receive my message ?

Cordialement,
Benjamin AIMARD

Le 12/03/2022 à 09:52, digitalrevisor a écrit :

**Comment by digitalrevisor** [comment]
How do i that?

Roger

lør. 12. mar. 2022, 09:54 skrev Benjamin ***@***.***>:

**Comment by digitalrevisor** [comment]
How do I do that?

Roger

lør. 12. mar. 2022, 09:54 skrev Benjamin ***@***.***>:

**Reply by baimard** [reply]
Do you use docker or a physical serveur ?

**Comment by digitalrevisor** [comment]
Yes I confirm this.

But I think its something about the Nextclout theme we use. We dont have
this issue on other apps in Nextcloud.

Is that something you can customize in the code? That it can work on every
Themes.
We have a modified theme.

Med vennleg helsing

Roger Hopland

Statsautorisert revisor

Autorisert regnskapsfører


Digital Revisor AS


Naustdalsvegen 1B, 6800 Førde | Avdelinger - Bergen | Nordfjordeid | Måløy
*S:* +47 57824090 | *M:* +47 90077351 | *E:* ***@***.*** | *W:*
https://jointlyoffice.com/



tir. 15. feb. 2022 kl. 04:52 skrev Benjamin ***@***.***>:

**Reply by baimard** [reply]
I need an acces to your nextcloud to analyse. if it's possible

**Comment by digitalrevisor** [comment]
Ok. Sign up to jointlyoffice.com and test.

Med vennleg helsing

Roger Hopland

Statsautorisert revisor

Autorisert regnskapsfører


Digital Revisor AS


Naustdalsvegen 1B, 6800 Førde | Avdelinger - Bergen | Nordfjordeid | Måløy
*S:* +47 57824090 | *M:* +47 90077351 | *E:* ***@***.*** | *W:*
https://jointlyoffice.com/



man. 28. mar. 2022 kl. 09:17 skrev Benjamin ***@***.***>:

**Reply by baimard** [reply]
have you receive my email ?

**Comment by digitalrevisor** [comment]
See new file created today

tir. 15. feb. 2022 kl. 04:52 skrev Benjamin ***@***.***>:

> Very strange. Have you clear your cache ?
>
Yes

> Version of nextcloud ?
>
25.0.2

> Is the application work ?
>
Yes ok.

> —
> Reply to this email directly, view it on GitHub
> <https://github.com/baimard/gestion/discussions/165#discussioncomment-2177956>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOJTWUSYJNBMUSBIJSHC7C3U3HEZNANCNFSM5OMX63PA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

===client===
1;Hopland;Roger;;Hopland ***@***.******@***.***
2;Oos;Paal;;DR ***@***.******@***.***
3;tester;henrik;some ***@***.***;some ***@***.***
===devis===
***@***.***;comment;1.0;New
***@***.***;commentaire ?;0.1;Création
***@***.***;commentaire ?;0.1;Création
***@***.***;Comment;0.1;New
***@***.***;Comment;0.1;New
===facture===
1;1;15.10.2021;2021-10-06;Means of ***@***.***;NC;1.0
2;2;Texte libre;2021-10-06;Means of ***@***.***;NC;1.0
3;3;Texte libre;2021-11-12;Means of ***@***.***;NC;1.0
4;4;Texte libre;2021-11-18;Means of ***@***.***;NC;1.0
5;Invoice number;Text free;2021-11-24;Means of ***@***.***;NC;1.0
6;Invoice number;Text ***@***.***;NC;1.0
7;Invoice number;Text free;2021-11-30;Means of ***@***.***;NC;1.0
8;INVOICE-8;Text free;2022-03-29;Means of ***@***.***;NC;1.0
9;INVOICE-9;Text free;2022-03-29;Means of ***@***.***;NC;1.0
10;INVOICE-10;Text free;2022-05-30;Means of ***@***.***;NC;1.0
11;INVOICE-11;Text free;2022-10-29;Means of ***@***.***;NC;1.0
===produit===
1;15001;Revisjon og ***@***.***;0
***@***.***;0
***@***.***;0
***@***.***;0
***@***.***;0
***@***.***;0
***@***.***;0
***@***.***;0
***@***.***;0
***@***.***;0
***@***.***;0
***@***.***;0
***@***.***;0
***@***.***;0
===produit_devis===
***@***.***;0;
***@***.***;0;
***@***.***;0;
***@***.***;0;
***@***.***;0;
***@***.***;0;
***@***.***;0;-
***@***.***;0;-
===configuration===
1;Revisorgruppen Fjordane ***@***.***;004790077351;6800 F&oslash;rde;//1 a-testfolder &aelig;-&aring;-&oslash; &AElig; - &Aring; - ***@***.***;Betales til kontonummer: 4444.22.55555;25;8;NOK;1;en-EN
2;Your company name;Your company contact surname;Your company contact name;;;Your company email;Your company phone;Your company ***@***.***;All Legal mentions, disclaimer or everything, you want place in the footer.;0;0;EUR;1;en-EN
3;Your company name;Your company contact last name;Your company contact first name;Company legal information line one;Company legal information line two;Your company email;Your company phone;Your company ***@***.***;;0;8;EUR;1;en-EN
4;Your company name;Your company contact last name;Your company contact first name;Company legal information line one;Company legal information line two;Your company email;Your company phone;Your company ***@***.***;All Legal mentions, disclaimer or everything you want to place in the footer.;0;8;EUR;1;en-EN
